roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -56,12 +56,12 @@
     create: true
   with_items:
     - regex: '^HTTP_PROXY='
-      line: "HTTP_PROXY={{ openshift.common.http_proxy }}"
+      line: "HTTP_PROXY={{ openshift.common.http_proxy | default('') }}"
```

Comment:
Does this fix the issues reported in https://bugzilla.redhat.com/show_bug.cgi?id=1386654